### DISCUSSION TITLE
AM not allowing running it with sudo

### DISCUSSION THREAD

**Discussion by WitHol** [ivan-hc/AM#1260] [Q&A]
I'm just a little curios, what the reason behind not being able to run am with sudo is.

**Comment by Samueru-sama** [comment]
> I'm just a little curios, what the reason behind not being able to run am with sudo is.

https://github.com/ivan-hc/AM/issues/826

**Comment by ivan-hc** [comment]
hi @WitHol , `sudo`/`doas` is set into a builtin environment variable, so you can use the command without adding `sudo`, also to save time.

Since we use an env variable, AM can be also used as AppMan, that instead does not require privileges and only installs apps locally.
